packages/hmr-plugin/src/actions/hmr-before-destroy.action.ts

```diff
@@ -0,0 +1,9 @@
+import { NgxsHmrSnapshot } from '../symbols';
+
+export class HmrBeforeDestroyAction {
+  static get type() {
```

Comment:
Please add this disclaimer above the return statement:
```ts
// NOTE: Not necessary to declare the type in this way in your code. See https://github.com/ngxs/store/pull/644#issuecomment-436003138
```

Reply:
I've seen you changed that to a property.
You need to change all the actions to the previous definition, please read the PR:
https://github.com/ngxs/store/pull/644